src/amo/css/inc/vars.scss

```diff
@@ -16,7 +16,7 @@ $breakpoints: (
   // This breakpoint exists for tweaking of very small screens.
     extraSmall: only screen and
     (
-      max-width: 349px,
+      max-width: 390px,
```

Comment:
Yep, I don't think we should be using it though (and not change it either)

Reply:
Okay! Thanks :) @willdurand,
I am changing it back to `349px`.